cdc/owner.go

```diff
@@ -1626,3 +1633,9 @@ func (o *Owner) startCaptureWatcher(ctx context.Context) {
 		}
 	}()
 }
+
+// reset gcBLT
+func (o *Owner) setGcBLT(gcBLT int64) {
+	o.gcBLT = gcBLT
```

Comment:
Is there any data race here?

Reply:
In `http_handler.go` the caller function `handleConfigGCTimeToBlock` must acquire  a write lock before calling the function, thus avoiding data race here.